app-misc/pet/pet-0.3.0.ebuild

```diff
@@ -46,7 +49,7 @@ src_install() {
 
 	if use zsh-completion ; then
 		insinto /usr/share/zsh/site-functions
-		doins src/github.com/knqyf263/pet/misc/completions/zsh/_${PN}
+		doins src/github.com/knqyf263/pet/misc/completions/zsh/_"${PN}"
```

Comment:
This doesn't belong in this commit. That said, it's unnecessary to quote it here.

Reply:
A whole "${PN}" replaced by pet.